README.md

```diff
@@ -15,119 +15,9 @@ experience to Ruby developers using modern standards for cross-editor features,
-documentation](SEMANTIC_HIGHLIGHTING.md).
-
-### Configuring code indexing
-
-By default, the Ruby LSP indexes all Ruby files defined in the current project and all of its dependencies, including
-default gems, except for
-
-- Gems that only appear under the `:development` group
-- All Ruby files under `test/**/*.rb`
-
-This behaviour can be overridden and tuned. Learn how to configure it [for VS Code](vscode/README.md#Indexing-Configuration) or [for other editors](EDITORS.md#Indexing-Configuration).
-
-Note that indexing-dependent behavior, such as definition, hover, completion or workspace symbol will be impacted by
-the configuration changes.
-
-The older approach of using a `.index.yml` file has been deprecated and will be removed in a future release.
-
-```yaml
-# Exclude files based on a given pattern. Often used to exclude test files or fixtures
-excluded_patterns:
-  - "**/spec/**/*.rb"
-
-# Include files based on a given pattern. Can be used to index Ruby files that use different extensions
-included_patterns:
-  - "**/bin/*"
-
-# Exclude gems by name. If a gem is never referenced in the project's code and is only used as a tool, excluding it will
-# speed up indexing and reduce the amount of results in features like definition or completion
-excluded_gems:
-  - rubocop
-  - pathname
-
-# Include gems by name. Normally used to include development gems that are excluded by default
-included_gems:
-  - prism
-```
-
-### Addons
-
-The Ruby LSP provides an addon system that allows other gems to enhance the base functionality with more editor
-features. This is the mechanism that powers addons like
-
-- [Ruby LSP Rails](https://github.com/Shopify/ruby-lsp-rails)
-- [Ruby LSP RSpec](https://github.com/st0012/ruby-lsp-rspec)
-- [Ruby LSP rubyfmt](https://github.com/jscharf/ruby-lsp-rubyfmt)
-
-Additionally, some tools may include a Ruby LSP addon directly, like
-
-- [Standard Ruby (from v1.39.1)](https://github.com/standardrb/standard/wiki/IDE:-vscode#using-ruby-lsp)
-
-Other community driven addons can be found in [rubygems](https://rubygems.org/search?query=name%3A+ruby-lsp) by
-searching for the `ruby-lsp` prefix.
-
-For instructions on how to create addons, see the [addons documentation](ADDONS.md).
-
-## Learn More
-
-* [RubyConf 2022: Improving the development experience with language servers](https://www.youtube.com/watch?v=kEfXPTm1aCI) ([Vinicius Stock](https://github.com/vinistock))
-* [Remote Ruby: Ruby Language Server with Vinicius Stock](https://remoteruby.com/221)
-* [RubyKaigi 2023: Code indexing - How language servers understand our code](https://www.youtube.com/watch?v=ks3tQojSJLU) ([Vinicius Stock](https://github.com/vinistock))
+Please refer to the official [documentation](https://shopify.github.io/ruby-lsp) for more information about [installation and usage](https://shopify.github.io/ruby-lsp#usage) and [supported features](https://shopify.github.io/ruby-lsp#features).
```

Comment:
I wonder if we should still keep at least the basics of usage in the readme, pointing people to the VS Code extension and the more extensive documentation.

Reply:
I lean towards to leading users to the official website from the very beginning, especially with the one-page design.
This way, users could immediately learn and explore other content like feature demos, or at least know there presence, when they start using Ruby LSP.

Reply:
We can certainly cut down a lot of the content in the README but I do think we should keep a minimal explanation of what Ruby LSP does and how to install it (for VS Code at least).

Reply:
I added basic usage instructions back to readme.